tests/integration-tests/tests/dashboard/test_dashboard/test_dashboard/pcluster.config.yaml

```diff
@@ -0,0 +1,40 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    VpcId: {{ vpc_id }}
```

Comment:
VpcId doesn't exist in the new configuration file. We're calculating it from the subnet id.

Reply:
Removed the "VpcId"